### PR TITLE
Lel/update face sdk

### PR DESCRIPTION
**Pull request by leareai**
# Description
Some SDK APIs are still under review and need more discussion to get better design.
In order not to block SDK for detect livenss, we decide to release SDK with part of the APIs

In this PR, we
1. Update TypeSpec to
1-1. Reflect the API removals
1-2. Adjust the customization according to new TypeSpec change
2. Remove corresponding sample codes of documentations.

# All SDK Contribution checklist:
- [X] **The pull request does not introduce [breaking changes]**
- [X] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [X] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/face/azure-ai-vision-face/src/main/java/com/azure/ai/vision/face/models/FaceAttributeType.java [hunk 1]
```diff
@@ -181,17 +180,21 @@ public static final class Detection01 {
...
+        /**
+         * Face is blurry or not. Level returns 'Low', 'Medium' or 'High'. Value returns a number between [0,1], the
+         * larger the blurrier.
+         */
+        public static final FaceAttributeType BLUR = FaceAttributeType.BLUR;
+
         /**
          * 3-D roll/yaw/pitch angles for face direction.
          */
-        @Generated
```
Comment (weidongxu-microsoft) [inline sdk/face/azure-ai-vision-face/src/main/java/com/azure/ai/vision/face/models/FaceAttributeType.java:187]: why is this?

File: sdk/face/azure-ai-vision-face/src/main/resources/META-INF/azure-ai-vision-face_apiview_properties.json
Comment (weidongxu-microsoft) [inline sdk/face/azure-ai-vision-face/src/main/resources/META-INF/azure-ai-vision-face_apiview_properties.json:1]: There is lots of model that no longer generated from typespec. Delete them in models / implementation.models package.

You may want to delete the 2 model folders and regen.

Comment (leareai) [reply sdk/face/azure-ai-vision-face/src/main/resources/META-INF/azure-ai-vision-face_apiview_properties.json:1]: Updated. Thanks!!

File: sdk/face/azure-ai-vision-face/src/main/java/com/azure/ai/vision/face/models/FaceAttributeType.java (hunk 1 above)
Comment (leareai) [reply sdk/face/azure-ai-vision-face/src/main/java/com/azure/ai/vision/face/models/FaceAttributeType.java:187]: We actually support this attributes for detection03, but our documentation wasn't updated. So I missed this attributes. We discover it in recent documentation migration.

Comment (weidongxu-microsoft) [reply sdk/face/azure-ai-vision-face/src/main/java/com/azure/ai/vision/face/models/FaceAttributeType.java:187]: Ah, all this is added by hand?

Comment (leareai) [reply sdk/face/azure-ai-vision-face/src/main/java/com/azure/ai/vision/face/models/FaceAttributeType.java:187]: Yes.

**Review by weidongxu-microsoft — APPROVED** [review]

**Pull request merged**